### ISSUE TITLE
What is the learning rate for the first stage of training the network?

### ISSUE THREAD

**Issue by plusgood-steven** [dasongli1/Learning_degradation#4]

**Comment by dasongli1** [comment]
The learning rate for the first stage is 2e-4. 
Thank you for your interest in learning degradation. The source code about learning degradation (pre-training) will be released soon.
